std_msgs/String[] string_array